{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py

```diff
@@ -39,5 +39,155 @@ def build_artifacts(location: str, output_dir: str, append: bool, verbose: int):
         How noisy the logger should be.
 
     """
-    # Stubbed out pending further consideration of best location for this code.
-    pass
+    output_dir = Path(output_dir)
+
+    if location in project_globals.LOCATIONS:
+        path = Path(output_dir) / f'{sanitize_location(location)}.hdf'
+
+        if not append:
+            delete_if_exists(path)
+
+        build_single_location_artifact(path, location)
+
+    elif location == 'all':
+        # FIXME: could be more careful
+        existing_artifacts = set([item.stem for item in output_dir.iterdir()
```

Comment:
Make a list of output paths, then pass it to delete if exists

Reply:
Done.

Reply:
Think we can replace all this logic with:
```
if location == 'all':
    paths = [output_dir / f'{sanitize_location(loc)}.hdf' for loc in project_globals.LOCATIONS]
    if not append:
        delete_if_exists(paths, confirm=True)
```

Reply:
Refactored this function.